node/CHANGELOG.md

```diff
@@ -15,10 +15,14 @@ All notable changes to this project will be documented in this file.  The format
 
 ### Added
 * Add `BinaryPort` interface along with the relevant config entries.
+* New seigniorage model that is independent of the details of consensus (compatible with both Highway and Zug) and based solely upon block proposals, signature generation and signature distribution by validators.
```

Comment:
I think this might go into the "Changed" section, because we removed the old model at the same time. So essentially, we changed how the rewards work.

Reply:
Alright, how about now @fizyk20